src/nvim/charset.c

```diff
@@ -1454,6 +1454,20 @@ char_u* skipdigits(char_u *q)
   return p;
 }
 
+/// skip over binary digits
+///
+/// @param q
+///
+/// @return Pointer to the character after the skipped digits.
+char_u* skipbin(char_u *q)
```

Comment:
Do not use `char_u` in new functions dealing with strings. This needs to be `const char *skipbin(const char *q)`.